app/assets/javascripts/application.js

```diff
@@ -17,6 +17,7 @@
 //= require jquery-ui/widgets/sortable
 //= require jquery.remotipart
 //= require d3
+//= require c3
```

Comment:
Both d3 and c3 here are duplicates. Since you're already importing c3 in members / targets charts, you shouldn't need it here.